eq-author/src/storybook/Components/Radio.stories.mdx

```diff
@@ -0,0 +1,166 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+
+import { Input, Label } from "components/Forms";
+
+import { colors } from "constants/theme";
+
+<Meta title="Components/Radio button" component={Input} />
+
+# Radio button
+
+Lets users select one option from a selection of multiple different options.
+
+## Example
+
+<Canvas>
+  <Story name="Default">
+    <Label htmlFor="optionOne">
+      <Input type="radio" id="optionOne" name="option" value="optionOne" />
+      Option one
+    </Label>
+    <Label htmlFor="optionTwo">
+      <Input type="radio" id="optionTwo" name="option" value="optionTwo" />
+      Option two
+    </Label>
+    <Label htmlFor="optionThree">
+      <Input type="radio" id="optionThree" name="option" value="optionThree" />
+      Option three
+    </Label>
+  </Story>
+</Canvas>
+
+## Variants
+
+<Canvas>
+  <Story name="Radio Box">
+    <Label htmlFor="radioBoxOptionOne" style={{
+        padding: '2.5em 1.5em',
+        'border-radius': '1px solid ${colors.bordersLight}',
+        border: '1px solid ${colors.bordersLight}',
+        flex: '1 1 33.3333333%',
+        cursor: 'pointer',
+        display: 'flex',
+        background: '#fff',
+        color: '${colors.textLight}',
+        position: 'relative',
+        transition: 'padding 200ms ease-in-out',
+        'flex-direction': 'column',
+        'box-shadow': '0 2px 2px 0 rgba(0, 0, 0, 0.18)',
+        'border-radius': '0.25em',
+        'margin-bottom': '1.25em',
```

Comment:
Do you think it would be possible to separate this styling into a component in the story? It's reused for each `Label`. Could also do the same for the styling of the `<span>`'s